src/components/cl/hier/alltoallv/alltoallv.c

```diff
@@ -140,11 +144,16 @@ UCC_CL_HIER_PROFILE_FUNC(ucc_status_t, ucc_cl_hier_alltoallv_init,
         return UCC_ERR_NOT_SUPPORTED;
     }
 
-    if (!SBGP_ENABLED(cl_team, NODE) || !SBGP_ENABLED(cl_team, FULL)) {
-        cl_debug(team->context->lib, "alltoallv requires NODE and FULL sbgps");
+    if (!SBGP_ENABLED(cl_team, FULL)) {
+        cl_debug(team->context->lib, "alltoallv requires FULL sbgps");
         return UCC_ERR_NOT_SUPPORTED;
     }
 
+    if (!SBGP_ENABLED(cl_team, NODE)) {
```

Comment:
Minor comment: I think we can save one branching by merging this "if" with the one in line 178.